Alexandra/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Tea shop</title>
+</head>
+
+<body>
+    <div class="container">
+        <nav id="topBar">
+            <b> Trending deals </b> &nbsp;&nbsp; <a class="topbar" href="https://joesteacompany.com/">See all Deals</a>
+        </nav>
+        <div class="top-container">
+            <div class="tea row">
+                <div class="teacolumn">
+                    <p class="top">Queen of green</p>
+                    <img src="https://joesteacompany.com/wp-content/uploads/2015/08/QUEEN-OF-GREEN-600x600-1.jpg">
+                    <p><span class="discount">Up to 15% </span> &nbsp; <span class="end">Ends in 01:26:16</span></p>
+
+                </div>
+
+                <div class="teacolumn">
+                    <p class="top">Sweet camolile</p>
+                    <img src="https://joesteacompany.com/wp-content/uploads/2015/08/SWEET-CHAMOMILE-600x600-1.jpg">
+                    <p><span class="discount">Up to 21%  </span> &nbsp;<span class="end">Ends in 02:21:16</span></p>
+
+                </div>
+                <div class="teacolumn">
+                    <p class="top">Whiter than white</p>
+                    <img src="https://joesteacompany.com/wp-content/uploads/2015/08/WHITER-THAN-WHITE-600x600-1.jpg">
+                    <p><span class="discount">Up to 28% </span> &nbsp;<span class="end">Ends in 01:11:16</span></p>
+                </div>

```

Comment:
The structure is good. Try to match it as much as possible with what we have as a template and example